nfc/labAttendancy.py

```diff
@@ -34,14 +38,53 @@ def on_connect(tag):
 
     sc = nfc.tag.tt3.ServiceCode(service_code >> 6, service_code & 0x3f)
     bc0 = nfc.tag.tt3.BlockCode(0, service=0)
+
+
+
     try:
         id_data = tag.read_without_encryption([sc], [bc0])
+    except Exception:
+        print("touch it again")
+        # print("showwarning",messagebox.showwarning("LabAttendancy","Touch it again!"))
+    else:
         id = id_data[2:13].decode("utf-8")
         print("Student Num: " + id)
-    except:
-        print("touch it again")
-
+        msg_info(id)
```

Comment:
ここでdoPost()を呼び出していないのでPOSTが行われていない
ここは自分もGitに載せてなくてごめん

```suggestion
        postData(id)
        msg_info(id)
```